internal/integration/oidc.go

```diff
@@ -227,17 +228,25 @@ func GetRequest(url string, headers map[string]string) (*http.Request, error) {
 }
 
 func CheckRedirect(req *http.Request) (*url.URL, error) {
-	client := &http.Client{
+	c := &http.Client{
```

Comment:
`client` already exists in the scope as a pkg name, just fixing some linter warning